hbase_sql/src/main/java/com/splicemachine/derby/stream/spark/SparkScanSetBuilder.java

```diff
@@ -113,7 +120,19 @@ else if (storedAs.equals("O"))
         if (oneSplitPerRegion) {
             conf.set(MRConstants.ONE_SPLIT_PER_REGION, "true");
         }
-        if (useSample) {
+        if (dsp.isSparkExplain()) {
```

Comment:
I'd either remove these blocks or make them return an empty dataset

Reply:
OK, I've reverted the changes.